src/systems/systemsTableView.component.tsx

```diff
@@ -35,11 +36,12 @@ export const SystemsTableView = (props: SystemsTableViewProps) => {
   );
 
   const noResultsText = 'No systems found';
-  const columns = React.useMemo<MRT_ColumnDef<System>[]>(
-    () => [
+  const columns = React.useMemo<MRT_ColumnDef<System>[]>(() => {
+    return [
```

Comment:
```suggestion
const columns = React.useMemo<MRT_ColumnDef<System>[]>(
    () => [
```

Reply:
This just errors out, I think the return is needed as it is multiple objects (columns) now. Like in all the other table files

Reply:
It works if I use
```
  const columns = React.useMemo<MRT_ColumnDef<System>[]>(
    () => [
      {
        header: 'Name',
        accessorKey: 'name',
        size: 400,
        Cell: ({ renderedCellValue, row }) => {
          const canPlaceHere =
            type === 'copyTo' || !selectedSystemIds.includes(row.original.id);
          return (
            <Typography
              sx={{
                color: canPlaceHere ? 'inherit' : 'action.disabled',
              }}
            >
              {renderedCellValue}
            </Typography>
          );
        },
      },
      {
        header: 'Last modified',
        accessorFn: (row) => new Date(row.modified_time),
        id: 'modified_time',
        filterVariant: 'datetime-range',
        size: 400,
        enableGrouping: false,
        Cell: ({ row }) =>
          row.original.modified_time &&
          formatDateTimeStrings(row.original.modified_time, true),
      },
    ],
    [selectedSystemIds, type]
  );
```

Reply:
ahhh, I'd forgotten to change the semicolon after the closing bracket to a comma